qiskit_experiments/test/mock_iq_helpers.py

```diff
@@ -0,0 +1,393 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Probability and phase functions for the mock IQ backend."""
+
+from abc import abstractmethod
+from typing import Dict, List
+import numpy as np
+from qiskit import QuantumCircuit
+from qiskit.providers.aer import AerSimulator
+
+
+class MockIQExperimentHelper:
+    """Abstract class for experiment tools"""
+
+    @abstractmethod
+    def compute_probability(self, circuits: List[QuantumCircuit]) -> List[Dict[str, float]]:
```

Comment:
Isn't `compute_probabilities` a abetter name?

Reply:
Yeas, I think so to.
Changing as suggested.